buildSrc/src/main/java/quilt/internal/WikiBuildPlugin.java

```diff
@@ -39,6 +40,28 @@ public void apply(Project target) {
 		target.getTasks().register("generateWikiTree", GenerateWikiTreeTask.class);
 		target.getTasks().register("generateWiki", GenerateWikiTask.class);
 		target.getTasks().register("testWiki", TestWikiTask.class);
+		target.allprojects(project -> {
+			var extension = project.getExtensions().create("wikiMappings", MappingTranslationExtension.class);
+			if (project.hasProperty("minecraft_version")) {
+				String minecraftVersion = project.findProperty("minecraft_version").toString();
+				extension.getMinecraftVersion().set(project.findProperty("minecraft_version").toString());
+				extension.getMojmapTranslatable().set(true);
+			}
+			if (project.file("src/main/java").exists()) {
+				project.getTasks().register("translateToMojmaps", MigrateMappingsTask.class).configure(task -> {
```

Comment:
Since we also have yarn and qm as available options, should we also have tasks to generate them? Or, should we state the language and default mapping set for that language and say we only support remapping to mojmaps?

Reply:
Currently, only Mojmaps; yarn is not supported by the mapping switching, so there's no point to generation anything for it, and the assumption is that folks will write in QM and translate to Mojmaps, I think. Doing it both ways causes all sorts of issues